vuepress/.gitignore

```diff
@@ -19,3 +19,4 @@ yarn-error.log*
 *.njsproj
 *.sln
 *.sw?
+/docs/.vuepress/dist/
```

Comment:
Is this leading slash correct here?

Reply:
I think it's a little more efficient since it anchors to the directory the .gitignore is located in, e.g. https://stackoverflow.com/questions/24139478/when-to-use-leading-slash-in-gitignore. That being said, I use IntelliJ and I just used its Git -> Add to .gitignore option which means I pretty much ignore what it spits out since it's always worked for me. :)

Reply:
Sounds good. Works for me